lib/hermetic/slash.ex

```diff
@@ -0,0 +1,152 @@
+alias Hermetic.{YouTrack, Attachment, Slack}
+
+defmodule Hermetic.Slash do
+  @moduledoc """
+  Handles Slack slash commands
+  """
+
+  import Plug.Conn
+
+  use Plug.ErrorHandler
+
+  def init([]), do: []
+
+  @doc """
+  Return usage string given a /command
+  """
+  @spec usage(String.t()) :: String.t()
+  def usage(command) do
+    case command do
+      "/yt-add" -> "projectid [@assignee] [#tag] Title text"
+      "/yt-cmd" -> "issue-id command"
+    end
+  end
+
+  def handle_errors(conn, %{kind: _kind, reason: _reason, stack: _stack}) do
+    send_resp(conn, 200, "Usage: " <> usage(conn.body_params["command"]))
+  end
+
+  @doc """
+  Split off the first word based on whitespace.
+  Returns a tuple of the first word and the remainder.
+  """
+  @spec split_word(String.t()) :: [String.t()]
+  def split_word(string) do
+    string
+    |> String.trim_leading()
+    |> String.split(~r/\s+|$/, parts: 2)
+  end
+
+  @doc ~S"""
```

Comment:
Does that require `~S"""`? (not sure)

Reply:
Probably not in this case, but I think it's better to use any time you put code in a string. You don't want it to parse escape codes if it encounters any.

Reply:
OK, let's let's substitute `@doc """` for `@doc ~S"""` everywhere then.